src/iuse.cpp

```diff
@@ -3072,6 +3146,8 @@ int iuse::noise_emitter_on(player *p, item *it, bool t, const tripoint &pos)
 
 int iuse::ma_manual(player *p, item *it, bool, const tripoint& )
 {
+    // [CR] - should NPCs just be allowed to learn this stuff? Just like that?
```

Comment:
On that note, maybe we could stretch it out by requiring you to read the book for a variable duration of time, similar to leveling a skill via book.